WireDomain/Sources/WireDomain/Synchronization/SyncManager.swift

```diff
@@ -32,20 +38,103 @@ protocol SyncManagerProtocol {
 
 }
 
+public struct MLSProvider {
+    let service: any MLSServiceInterface
+    let isMLSEnabled: Bool
+
+    public init(
+        service: any MLSServiceInterface,
+        isMLSEnabled: Bool
+    ) {
+        self.service = service
+        self.isMLSEnabled = isMLSEnabled
+    }
+}
```

Comment:
It'd be nice to move this to a new file.

Reply:
done in https://github.com/wireapp/wire-ios/pull/1999/commits/25021b3841574291cb480d46d50180fec40f9654